### PR TITLE
Add Tackle2 export to CLI tool

### PR DESCRIPTION
**Pull request by aufi**
Adding Tackle 2 API objects export to tackle CLI tool and few minor updates on the tool.

The Tackle 2 export scenario added to README:
```
### Export and Import data between Tackle 2 instances

This feature focuses on application inventory, but exports and imports most of Tackle 2 objects except TaskGroups&Tasks and Keycloak identitty provider data.

Export/imported entities:
- Applications, Dependencies, Assessments, Reviews, Tag Types, Tags, Job Functions, Stakeholder Groups, Stakeholders, Business Services, Proxies, Identities (sensitive data are encrypted with ```encryption_passphase``` config file field)
- Application's Buckets data (e.g. app binaries or windup reports)

### CODE REVIEW

**Comment by aufi** [comment]
WIP, adding buckets data.

File: hack/tool/tackle [hunk 1]
```diff
@@ -134,7 +145,7 @@ def cmdWanted(args, action):
 
 class TackleTool:
     # TYPES order matters for import/upload to Tackle2
-    TYPES = ['tagtypes', 'tags', 'jobfunctions', 'stakeholdergroups', 'stakeholders', 'businessservices', 'applications', 'proxies', 'dependencies', 'assessments', 'assessments--assessment-risk', 'assessments--confidence', 'reviews', 'proxies', 'identities']
```
Comment (aufi) [inline hack/tool/tackle]: 'assessments--assessment-risk', 'assessments--confidence' types were left from past, it is not needed - such data are calculated from the main assessment questionnaire. (and these types collections were empty even in 2.0 version release)

File: hack/tool/tackle [hunk 2]
```diff
@@ -374,6 +440,177 @@ class Tackle12Import:
...
+            id.password    = id2['password']
+            id.key         = id2['key']
+            id.settings    = id2['settings']
+            self.add('identities', id)
+
+    def dumpTackle2Buckets(self):
+        for bucket in self.data['buckets']:
+            debugPrint("Downloading bucket data for %s from owner %s" % (bucket['path'], bucket['owner']))
+            bucketListing = apiRaw(self.tackle2Url + "/hub/" + bucket['owner'] + "/bucket/", self.tackle2Token)
+            bucketEntries = re.findall(("href=\"([^\"]+)\""), bucketListing)
```
Comment (aufi) [inline hack/tool/tackle:606]: Not sure if there not a better way to list bucket content

Comment (aufi) [reply hack/tool/tackle:606]: Converting to draft until ```buckets``` ednpoint is implemented in API and related code is updated.

**Comment by aufi** [comment]
This PR doesn't cover support for buckets API, which will be added in upcoming PRs.

**Comment by aufi** [comment]
Added a manifest file to the exort data directory. The file contains "known_string" to allow decryption check before running import and few other debugging information.

Example:
```
{
    "timestamp_utc": "2023-01-24T12:20:21.372082",
    "command_options": {
        "action": [
            "export"
        ],
        "config": "./tackle-config.yml",
        "data_dir": "./tackle-data",
        "verbose": true,
        "skipDestCheck": false,
        "disableSslWarnings": false,
        "ignoreImportErrors": false,
        "noAuth": false,
        "skipBuckets": false
    },
    "known_string": "tackle-cli-DFRWrubD5wkCr82gro5m",
    "crypted_known_string": "5Ik8XLmsagNGFJYsL6dtCSCUdPjK3j+9pdBrsGSEshg6exBd01nr0fwk09QtEtE="
}
```

**Comment by aufi** [comment]
The DCO check failure is caused by commits in this PR created before the signing policy was introduced, so I believe it should be OK.

**Review by jortel — APPROVED** [review]
LGTM

**Pull request merged**